src/plugins/sorting-task/SortingTaskNode/ReadView/style.scss

```diff
@@ -0,0 +1,79 @@
+.learning-items{
+  padding: 40px; /* padding to prevent overflow during animation*/
+
+  --border-width: 2px;
+  --border-radius: 6px;
+}
+
+/* inspiration for card-flip animation: https://www.w3schools.com/howto/howto_css_flip_card.asp */
+/* The flip card container - set the width and height to whatever you want. We have added the border property to demonstrate that the flip itself goes out of the box on hover (remove perspective if you don't want the 3D effect */
+.flip-card {
+  background-color: transparent;
+  width: 100%;
+  max-width: 300px;
+  height: 50vh;
+  max-height: 400px;
+  margin: 0 auto;
+  perspective: 1000px;
+}
+
+/* This container is needed to position the front and back side */
+.flip-card-inner {
+  position: relative;
+  width: 100%;
+  height: 100%;
+  text-align: center;
+  transition: transform 0.8s;
+  transform-style: preserve-3d;
+  border: var(--border-width) solid #333;
+  border-radius: var(--border-radius);
+}
+
+/* Do an horizontal flip when you move the mouse over the flip box container */
+/* TODO replace :hover with .flipped or .active class and toggle the class with js on click */
+.flip-card:hover .flip-card-inner {
+  transform: rotateY(180deg);
+}
+
+/* Position the front and back side */
+.flip-card-content {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  margin: 0 !important; /* "!important" to reset all margins applied by default to dt & dd */
+  backface-visibility: hidden;
+  border-radius: calc(var(--border-radius) - var(--border-width));
+  /* center the content */
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+
+  background-color: #fff;
+  color: #333;
+
+  .flip-card-title{
+    margin: 1em 0;
+    font-size: 1.2em;
+  }
+
+  p{
```

Comment:
I’d recommend not to nest selectors this way as it’ll introduce complexity with selector specificity. Here’s an article explaining it more in details.